execution/utility.sh

```diff
@@ -60,7 +60,7 @@ function outputLogEntry() {
 function willLog() {
   local severity="$1"
 
-  [[ ${LOG_LEVEL_ORDER[$(getLogLevel)]} -le ${LOG_LEVEL_ORDER[${severity}]} ]]
+  (( "${LOG_LEVEL_ORDER[$(getLogLevel)]}" <= "${LOG_LEVEL_ORDER[${severity}]}" ))
```

Comment:
Is this a refactor or was it not working?

Reply:
Thought it wasn't working, updated it, it didn't fix the issue I was having. It looked better so kept it

Reply:
It returns a boolean rather than a number which was why I was using `[[`. I'll merge